db.js

```diff
@@ -749,20 +770,29 @@ module.exports.CreateDB = function (parent, func) {
                         CREATE INDEX ndxsmbiostime ON smbios (time);
                         CREATE INDEX ndxsmbiosexpire ON smbios (expire);
                         `, function (err) {
-                            // Completed setup of SQLite3
+                            // Completed DB creation of SQLite3
+                            //WAL mode instead of roll-back/delete
+                            obj.file.run( 'PRAGMA journal_mode=WAL;' );
+                            //Together with the optimize in the maintenance run, see https://sqlite.org/pragma.html#pragma_optimize
+                            obj.file.run( 'PRAGMA optimize=0x10002;' ); 
                             setupFunctions(func);
                         }
                     );
                 });
                 return;
-            } else if (err) { console.log("SQLite Error: " + err); exit(1); return; }
+            } else if (err) { console.log("SQLite Error: " + err); process.exit(0); return; }
```

Comment:
this should exit with 1 because the was an error, if it exits with 0, the system thinks it was a normal close with no errors, which the wasnt!

Reply:
This was changed to gracefully exit, conforming to how it is done elsewhere in the code:
https://github.com/Ylianst/MeshCentral/blob/7928f7fb3039e62ffde71867ec3153a72a305c50/db.js#L97
With a '1' exit, the app tries to restart itself, with '0' not. This seems the proper way to exit as it is a fatal error, with an error message from the app.

Reply:
you are indeed correct, my mistake!
i just scanned for `exit(1)` and only this exists, while others are exit`(0)`
im having one of those days today i think,
i need some more sleep 💤